README.md

```diff
@@ -1,70 +1,176 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
```

Comment:
_note: Since there is no open PR in [the backend project ](https://github.com/huzaifakhan0308/full-stack-capstone-back-end)for me to review, I'll include my comments here for that project.  This review is based on the `API-documentation` branch which is the one with the most recent commits._

# Backend review

- [ ]  You are not including any unit tests, but the "test" folder still exists, which is used by a test framework. Kindly consider deleting that folder if you are not using it. 👍

- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪

- [ ] OPTIONAL: I recommend you include some unit tests, to ensure your app's integrity and quality.  Since this is not on the requirements, this is optional.